QUICKSTART.md

```diff
@@ -0,0 +1,308 @@
+OpenTelemetry offers four exporters out of the box:
+- In-Memory Exporter: keeps the data in memory, useful for debugging.
+- Jaeger Exporter: prepare and send the collected telemetry data to a Jaeger backend via gRPC.
+- Logging Exporter: saves the telemetry data into log streams.
+- OpenTelemetry Exporter: sends the data to the [OpenTelemetry Collector] (not yet implemented).
+
+[OpenTelemetry Collector]: https://github.com/open-telemetry/opentelemetry-collector
+
+```java
+tracerRegistry.addSpanProcessor(SimpleSpansProcessor.newBuilder(
+    InMemorySpanExporter.create()
+).build());
+tracerRegistry.addSpanProcessor(SimpleSpansProcessor.newBuilder(
+    new LoggingExporter()
+).build());
+
+ManagedChannel jaegerChannel = ManagedChannelBuilder.forAddress([ip:String], [port:int]).usePlaintext().build();
+JaegerGrpcSpanExporter jaegerExporter = JaegerGrpcSpanExporter.newBuilder()
+    .setServiceName("example").setChannel(jaegerChannel).setDeadline(30000)
+    .build();
+tracerRegistry.addSpanProcessor(BatchSpansProcessor.newBuilder(
+    jaegerExporter
+).build());
+```
+
+# Tracing
+
+In the following, we present how to trace code using the OpenTelemetry API.
+**Note:** Methods of the OpenTelemetry SDK should never be called.
+ 
+First, a `Tracer` must be acquired, which is responsible for creating spans and interacting with the [Context](#context-propagation).
+A tracer is acquired by using the OpenTelemetry API specifying the name and version of the library 
+instrumenting the instrumented library or application to be monitored. 
+More information is available in the specification chapter [Obtaining a Tracer].
+
+```java
+Tracer tracer = OpenTelemetry.getTracerRegistry().get("instrumentation-library-name","semver:1.0.0");
+```
+
+[Obtaining a Tracer]: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md#obtaining-a-tracer
+
+## Create basic Span
+To create a basic span, you only need to specify the name of the span.
+The start and end time of the span is automatically set by the OpenTelemetry SDK.
+```java
+Span span = tracer.spanBuilder("SpanName").startSpan();
+// your use case
+...
+span.end();
+```
+
+## Create nested Spans
+
+Most of the time, we want to correlate spans for nested operations.
+OpenTelemetry supports tracing within processes and across remote processes.
+For more details how to share context between remote processes, see [Context Propagation](#context-propagation).
+
+For a method `A` calling a method `B`, the spans could be manually linked in the following way:
+```java
+void a() {
+  Span parentSpan = tracer.spanBuilder("a").startSpan();
+  b(parentSpan);
+  parentSpan.end();
+}
+void b(Span parentSpan) {
+  Span childSpan = tracer.spanBuilder("b").setParent(parentSpan).startSpan();
+  // do stuff
+  childSpan.end();
+}
+```
+Since it is not always possible to change a method signature or rely on global variables, the OpenTelemetry API
```

Comment:
This line makes it sound like the example above is standard practice, instead of just here to demonstrate the mechanics of implicit context propagation.

Reply:
Agreed on this!